charts/steward/templates/clusterrole-tenant-controller.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: steward-tenant-controller
+  labels:
+    {{- include "steward.labels" . | nindent 4 }}
+rules:
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources: ["rolebindings"]
+  verbs: ["get","list","create"]
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources: ["clusterroles"]
+  verbs: ["bind"]
+  resourceNames:
+  {{- toYaml .Values.tenantController.possibleTenantRoles | nindent 4 }}
+- apiGroups: ["steward.sap.com"]
+  resources: ["tenants","tenants/status"]
+  verbs: ["get","list","patch","update","watch"]
+- apiGroups: [""]
+  resources: ["namespaces"]
```

Comment:
Is it possible to restrict namespace (delete) permissions only to namespaces where the controller is owner? Or is this already the case?

Reply:
This is not possible with K8s RBAC. You would have to list all those namespace here under `resourceNames`.